src/roles.js

```diff
@@ -17,25 +19,51 @@ export const LEFT_ROLE_ID = '8ced989b1904606b'
  */
 
 /**
- * @typedef {object} Capability
+ * @typedef {object} Role
+ * @property {RoleId} roleId
```

Comment:
```suggestion
 * @template {RoleId} [T=RoleId]
 * @typedef {object} Role
 * @property {T} roleId
```

See below

Reply:
Done in b8742e47508cdd69d24aeea067b38c7ee8ac3a91.